tools/autograd/derivatives.yaml

```diff
@@ -160,13 +177,20 @@
 # in Decalarations.yaml
 - name: abs(Tensor self) -> Tensor
   self: grad * self.sgn()
+  result: auto_element_wise
 
 - name: acos(Tensor self) -> Tensor
   self: grad * -((-self * self + 1).rsqrt()).conj()
 
 - name: add.Tensor(Tensor self, Tensor other, *, Scalar alpha=1) -> Tensor
   self: handle_r_to_c(self.scalar_type(), grad)
   other: handle_r_to_c(other.scalar_type(), maybe_multiply(grad, alpha.conj()))
+  result: maybe_multiply(self_t, alpha) + other_t
```

Comment:
This formula is still wrong.

Here are two proofs. First, proof by transposition of existing backwards formula:

```
[g_self, g_other] = [1, alpha.conj()] g
```

So the transposed formula is:

```
[g] = [[1], [alpha.conj()]] [self_t, other_t]
```

implying that the formula is `self_t + alpha.conj() * other_t`, which is flipped from what you have above. 

Second proof by differentials (ignoring complex numbers):

```
self + alpha * other
= (self_p + self_t + alpha * (other_p + other_t))
= (self_p + alpha * other_p) + (self_t + alpha * other_t)
```

still wrong.

Reply:
I'm not sure if we also need `handle_r_to_c`, maybe it is fine to not have it because the promotion happens automatically in forwards.

Reply:
Thanks for catching this!
I suspect that we don't have any test that checks gradients for alpha!=1 :/  (or at least there is none that I can find in common_method_invocation).